src/Bridge/Symfony/Bundle/SonataDoctrineBundle.php

```diff
@@ -13,13 +13,13 @@
 
 namespace Sonata\Doctrine\Bridge\Symfony\Bundle;
 
-use Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle;
+use Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle as SymfonySonataDoctrineBundle;
```

Comment:
Maybe something like
```suggestion
use Sonata\Doctrine\Bridge\Symfony\SonataDoctrineBundle as ForwardCompatibleSonataDoctrineBundle;
```
?